cmd/diagnostic.go

```diff
@@ -0,0 +1,16 @@
+package cmd
+
+import (
+	"github.com/spf13/cobra"
+)
+
+// diagnosticCmd represents the diagnostic command.
+var diagnosticCmd = &cobra.Command{
```

Comment:
I'm not opposed to this but i don't understand the reason for this subcommand. `pbnj diagnostic screenshot` vs `pbnj screenshot`, right? curious what you're thinking/optimizing for here?

Reply:
oh maybe because the protobuf service is called `diagnostic`?

Reply:
The idea is that in the future there may be other diagnostic commands other than screenshot - but it may be too early to start doing that; I can simplify it down to just screenshot.